### ISSUE TITLE
/etc/profile.d/pbench provides no environment variable pointing to the pbench installation root

### ISSUE THREAD

**Issue by RobertKrawitz** [distributed-system-analysis/pbench#1794]
It is not presently possible for any software running with the agent to know where the root directory is other than by knowing that it's currently hard-coded or by trying to parse PATH or PYTHONPATH.

**Comment by portante** [comment]
See also #1746

**Comment by ndokos** [comment]
In v0.69, this is `$(getconf.py pbench_install_dir pbench-agent)`. In 0.71 and later it is `$(pbench-config pbench_install_dir pbench-agent)`.  Is that enough?

**Comment by dbutenhof** [comment]
Of course, ironically, `pbench-config` requires that the full install path be in `PATH`, and that `_PBENCH_AGENT_CONFIG` is defined containing the full install path ... so if you already have the Pbench agent install path in your environment at least twice, you will be able to ask for it... ;-)

**Comment by RobertKrawitz** [comment]
> In v0.69, this is `$(getconf.py pbench_install_dir pbench-agent)`. In 0.71 and later it is `$(pbench-config pbench_install_dir pbench-agent)`. Is that enough?

Yes.

**Comment by webbnh** [comment]
> if you already have the Pbench agent install path in your environment at least three times, you will be able to ask for it... ;-)

But, when you install the Agent, it makes provision to place these automagically for you...so, as long as that's not defeated somehow, you just need to ask.  🙂

**Comment by pravins** [comment]
@ndokos what is present status of this?

**Comment by portante** [comment]
@pravins, this is work planned currently for the v0.73 release, but it should not be assigned to @ndokos until we determine who should do this work for that release.

**Comment by pravins** [comment]
18th oct, 2021: Seems during triaging or sprint planning it was unassigned to portante and assigned to ndokos.

**Comment by ndokos** [comment]
No change - I thought this was here as a reminder that we might need to do something when we move to adhere to Fedora packaging requirements. What that "something" is, is not clear to me at this time.
